contracts/voting/cwd-voting-cw721-staked/src/contract.rs

```diff
@@ -30,6 +26,7 @@ pub fn instantiate(
     info: MessageInfo,
     msg: InstantiateMsg,
 ) -> Result<Response<Empty>, ContractError> {
+    set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
```

Comment:
could we add a default migrate function where we set contract version as well?

Reply:
i'm actually in the camp of not having one. i feel having one that does nothing is more liable to be confusing than helpful. what we've seen is that migration code is actually non-trivial so probably not much benefit to cookie cutter ones. wdyt? if there's an advantage to having one, i'm happy to add it in.

Reply:
my argument for is that I remember working on a frontend change against the contracts some time back and realizing we had forgotten to call `set_contract_version` in the migrate functions so the frontend wasn't getting the updated version. I think it could be helpful to have a migrate function that sets version by default in every contract. the argument could also be made that you only need to add a migrate function when migration will start happening, but it's also easy to forget to add one when there aren't state migrations

Reply:
this is a good point. i'm going to go in the direction of having no migrate function, the ultimate reason being that having code that has no purpose is, imo, suboptimal (in our current circumstances, we'd never use it). let's keep an eye out for these kinds of things though moving forward!